docs/backends/quickwit.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: "Quickwit"
+---
+
+## Configuring Quickwit Backend
+
+The only required field is the Quickwit endpoint. This endpoint must be a valid URL for your Quickwit cluster.
```

Comment:
consider stating here that the url should be OTLP gRPC. You can check [Jaeger docs](https://github.com/keyval-dev/odigos/blob/415c65d865343e902768bc9fdef95dc1ee2356bd/docs/backends/jaeger.mdx#L10) for inspiration

Reply:
Done. I added more clarifications, hope it'll help to understand.